docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,152 @@
+
+
+== Process Engine
+
+=== Process and task administration API
+A process administration API and task administration API have been introduced to simplify some of the more complex administrator use cases.
+The process admin API enables you to:
+
+* Retrieve all process definition nodes
+* Cancel node instances
+* Retrigger node instances
+* Update the timer (absolute or relative)
+* List timer instances
+* Trigger nodes
+
+The task administration API enables you to:
+
+* Add and remove potential owners and excluded owners and business administrators
+* Add and remove task inputs and outputs
+* List, create, and cancel escalations and notifications
+
+=== Advanced task routing
+When tasks are assigned to a group of users, you can use pluggable task assignment strategies to automatically assign tasks to a suitable individual immediately. This enables more efficient task allocation, based on all properties associated with the task (for example potential owners and task priority but also task data that includes information such as geography, required skills, and so forth). You can use business rules to define the assignment logic, making it easy to customize this to your needs.
+
+=== Process execution server
+The process execution server (also known as kie-server) has been extended to support core engine features and offers a remote API for these operations. In addition the following architectural changes were introduced.
+
+==== Separate Business Central from execution server
+Business Central now delegates all of its requests to the execution server as well. The main advantage is that Business Central can now be used to monitor any set of execution servers. By linking the execution server to Business Central, the process and task monitoring UIs in Business Central can now connect to this execution server and show all relevant information. When multiple independent execution servers are used, you can either connect to a specific one or use the smart router to aggregate information across multiple servers. 
+
+==== Smart router
+The smart router (also known as kie-server-router) can be used as a proxy to help manage multiple independent process execution servers.
+
+=== Aliases
+You can now use aliases instead of container IDs in the remote REST APIs of the execution server. For more information, see the "Editing container aliases through Business Central" section of the _{PRODUCT} User Guide_.
+ 
+=== Business Central monitoring web application
+A new distribution `war` file has been added in this release. The goal is for the KIE Monitoring Web Application to be used along with Decision Server instances, managing all runtime capabilities. This includes managing containers, process instances, tasks, dashboards, and more. As opposed to the standard Business Central distribution, no authoring capabilities are offered.
```

Comment:
The intent is to only offer this in the context of cloud images, for on premise deployments we want customers to use the normal business-central instead.  This war is used in the cloud  images and customers can use the war for building custom images but it was decided not to offer it as an alternative to the business-central war for on premise deployments.  Probably might want to clarify this here.